pages/moderation.vue

```diff
@@ -93,4 +97,30 @@ watch(route, async () => {
 })
 
 await checkIfUserIsLoggedIn()
+
+function disableBackButton() {
```

Comment:
instead of fake pagination here, let's fix the real problem. 
When you click on a submission, we should do a router navigation to a new page based on the submission id. 

#### why?
- we won't need a hack anymore
- gives proper window history navigation
- we can actually send url links that will navigate directly to that submission
- keeping everything on `/moderation` is bound to cause other issues besides this one case

#### Is this a good time to do this? 
**yes**

Reply:
ok